generators/chipyard/src/main/scala/ChipTop.scala

```diff
@@ -12,7 +12,7 @@ import chipyard.iobinders.{IOBinders, TestHarnessFunction, IOBinderTuple}
 
 import barstools.iocell.chisel._
 
-case object BuildSystem extends Field[Parameters => RawModule]((p: Parameters) => Module(LazyModule(new DigitalTop()(p)).suggestName("system").module))
+case object BuildSystem extends Field[Parameters => LazyModule]((p: Parameters) => LazyModule(new DigitalTop()(p)).suggestName("system"))
```

Comment:
can we move the `suggestName` from here to `ChipTop` now?

Reply:
Also, is there ever a case where someone would want to have a concrete `Module` instead of a `LazyModule` as the system? It's easy to go from `LazyModule` -> `RawModule` but the reverse  requires a bit of work. Maybe that's rare enough that it's fine.

Reply:
Why is the reverse hard? It should be pretty easy to create a empty `LazyModule` to wrap an arbitrary concrete `Module`

Reply:
Eh, "Hard" is probably the wrong word. My point was that creating an entirely new class is more typing than 8 characters (`.module`).

Reply:
We could have both a buildTop and buildLazy top if we really wanted and delegate appropriately, or we could have the BuildTop function return `(Option[LazyModule], Module)`

Reply:
`(Option[LazyModule], Module)` actually seems pretty good.

Reply:
I think this might require more careful thought so i'll resolve it in a follow up PR.

Reply:
```suggestion
case object BuildSystem extends Field[Parameters => LazyModule]((p: Parameters) => LazyModule(new DigitalTop()(p)))
```